gifs_test.go

```diff
@@ -72,7 +72,7 @@ func TestImport(t *testing.T) {
 		t.Skip()
 	}
 
-	param := gifs.Request{
+	param := Request{
```

Comment:
Same thing here and below as commented here https://github.com/gifs/gifs_go/pull/5/files#r74696405.